contrib/mesos/pkg/election/etcd_master_test.go

```diff
@@ -22,13 +22,18 @@ import (
 	"github.com/coreos/go-etcd/etcd"
 	"k8s.io/kubernetes/pkg/tools"
 	"k8s.io/kubernetes/pkg/watch"
+	etcdtesting "k8s.io/kubernetes/pkg/storage/etcd/testing"
 )
 
 func TestEtcdMasterOther(t *testing.T) {
+	server := etcdtesting.NewEtcdTestClientServer(t)
+	defer server.Terminate(t)
+
 	path := "foo"
-	etcd := tools.NewFakeEtcdClient(t)
-	etcd.Set(path, "baz", 0)
-	master := NewEtcdMasterElector(etcd)
+	if _, err := server.Client.Set(path, "baz", 0); err != nil {
```

Comment:
I think we should, but the problem is that the EtcdMaster is taking EtcdClient (instead of storage.Interface).
So this is a bigger change, and I wanted to eliminate FakeEtcdClient dependency first.